### PR TITLE
Update python.rst

### PR DESCRIPTION
**Pull request by rohan472000**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: docs/apache-airflow/howto/operator/python.rst [hunk 1]
```diff
@@ -239,3 +239,11 @@ should return True when it succeeds, False otherwise. The other uses the Taskflo
     :dedent: 4
     :start-after: [START wait_function]
     :end-before: [END wait_function]
+
+You can also use the @task.sensor decorator from TaskFlow to define your sensor as a TaskFlow sensor.
+
+.. exampleinclude:: /../../airflow/example_dags/example_taskflow_sensor.py
```
Comment (pankajastro) [inline docs/apache-airflow/howto/operator/python.rst:245]: I think this example does not exist.

Comment (rohan472000) [reply docs/apache-airflow/howto/operator/python.rst:245]: I have made that file in other PR....look in pull request #30749

Comment (jedcunningham) [reply docs/apache-airflow/howto/operator/python.rst:245]: Can you just combine this change in that other PR then? Thanks.

Comment (rohan472000) [reply docs/apache-airflow/howto/operator/python.rst:245]: sure

Comment (rohan472000) [reply docs/apache-airflow/howto/operator/python.rst:245]: @jedcunningham @pankajastro check this PR where I've merged this into other one -> #30749

**Review by ankit-gautam23 — APPROVED** [review]
LGTM

File: docs/apache-airflow/howto/operator/python.rst (hunk 1 above)
Comment (pankajastro) [reply docs/apache-airflow/howto/operator/python.rst:245]: then please close this PR in favour of https://github.com/apache/airflow/pull/30749

**Pull request closed without merging**